tests/DatabaseSchemaManagerTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Tests;
+
+use Stancl\Tenancy\DatabaseManager;
+
+class DatabaseSchemaManagerTest extends TestCase
+{
+    public $autoInitTenancy = false;
+
+    /**
+     * Define environment setup.
+     *
+     * @param  \Illuminate\Foundation\Application  $app
+     * @return void
+     */
+    protected function getEnvironmentSetUp($app)
+    {
+        parent::getEnvironmentSetUp($app);
+
+        $app['config']->set('database.default', 'pgsql');
+        $app['config']->set('tenancy.database.based_on', null);
+        $app['config']->set('tenancy.database.suffix', '');
+        $app['config']->set('tenancy.using_schema_connection', true);
+    }
+
+    /** @test */
+    public function reconnect_method_works()
+    {
+        $old_connection_name = app(\Illuminate\Database\DatabaseManager::class)->connection()->getName();
+        
+        tenancy()->init('test.localhost');
+
+        app(\Stancl\Tenancy\DatabaseManager::class)->reconnect();
+        
+        $new_connection_name = app(\Illuminate\Database\DatabaseManager::class)->connection()->getName();
+
+        // $this->assertSame($old_connection_name, $new_connection_name);
+        $this->assertNotEquals('tenant', $new_connection_name);
```

Comment:
What's the reason for this?

Reply:
This test is copy from `DatabaseManagerTest` except this test `make_sure_using_schema_connection`, must i remove this?

Reply:
I resolved this in my PR to your repo.